From 5ec274215a8c12a623c0d20791c96831e3b6a961 Mon Sep 17 00:00:00 2001
From: alex <surkov.alex@gmail.com>
Date: Wed, 29 Apr 2015 12:13:57 +0300
Subject: [PATCH 3/3] PD#100201: ETH: get mac from sercurity key used version
 change nand3 to auto

Conflicts:
	drivers/amlogic/ethernet/am_net8218.c
---
 drivers/amlogic/ethernet/am_net8218.c |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/amlogic/ethernet/am_net8218.c b/drivers/amlogic/ethernet/am_net8218.c
index 3a3a533..b8d47af 100755
--- a/drivers/amlogic/ethernet/am_net8218.c
+++ b/drivers/amlogic/ethernet/am_net8218.c
@@ -906,8 +906,9 @@ void read_mac_from_nand(struct net_device *ndev)
 	u8 mac[ETH_ALEN];
 	char *endp;
 	int j;
-	extenal_api_key_set_version("auto3");
+//	extenal_api_key_set_version("auto3");
 	ret = get_aml_key_kernel("mac", print_buff, 0);
+	extenal_api_key_set_version("nand3");
 	printk("ret = %d\nprint_buff=%s\n", ret, print_buff);
 	if (ret >= 0) {
 		strcpy(ndev->dev_addr, print_buff);
-- 
1.7.10.4

